kpatch/kpatch

```diff
@@ -211,6 +271,11 @@ load_module () {
 		fi
 	done
 
+	if ! wait_for_patch_transition "$modname" ; then
+		unload_module "$modname"
+		die "failed to load module $module (stalled)"
```

Comment:
Here I think it should disable the patch, similar to the re-enable path above.

Reply:
unload_module() called in the error path will first disable the patch, then unload it.  Are you suggesting that we should only disable and not unload the module here?

Reply:
Ah, I see.  It would still be good to have an error message before calling unload_module(), like "failed to enable module $module (stalled), unloading"